my_list.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  attr_accessor :list
+
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] Good job working on this, but the MyList class should implement `#each` method that yields all members of `@list` in succession.